keyboards/miiiw/blackio83/rev_0100/rev_0100.c

```diff
@@ -0,0 +1,161 @@
+    {0, CS21_SW2, CS20_SW2, CS19_SW2}, /* RGB38 */
+    {0, CS21_SW3, CS20_SW3, CS19_SW3}, /* RGB39 */
+    {0, CS21_SW4, CS20_SW4, CS19_SW4}, /* RGB40 */
+    {0, CS21_SW5, CS20_SW5, CS19_SW5}, /* RGB41 */
+    {0, CS21_SW6, CS20_SW6, CS19_SW6}, /* RGB42 */
+    {0, CS21_SW7, CS20_SW7, CS19_SW7}, /* RGB43 */
+    {0, CS21_SW8, CS20_SW8, CS19_SW8}, /* RGB44 */
+    {0, CS21_SW9, CS20_SW9, CS19_SW9}, /* RGB45 */
+    {0, CS30_SW5, CS29_SW5, CS28_SW5}, /* RGB68 */
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1         C2        C3        C4        C5        C6        C7        C8          C9        C10         C11        C12        C13       C14*/
+		{  0,      1,         2,        3,        4,        5,        6,        7,        8,          9,        10,         11,       12,        13,       14 }, /* R0 */
+		{ 15,     16,        17,       18,       19,       20,       21,       22,       23,         24,        25,         26,       27,        28,       29 }, /* R1 */
+		{ 30,     31,        32,       33,       34,       35,       36,       37,       38,         39,        40,         41,       42,        NO_LED,   43 }, /* R2 */
+		{ 44,     45,        46,       47,       48,       49,       50,       51,       52,         53,        54,         55,       56,        NO_LED,   57 }, /* R3 */
+		{ 58,     59,        60,       NO_LED,   NO_LED,   61,       NO_LED,   NO_LED,   NO_LED,     62,        63,         64,       65,        66,       67 }, /* R4 */
+    }, {
+        {0, 0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},  {224, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16}, {224, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32}, {208, 32},
+		{0, 48},  {16, 48},  {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48}, {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48}, {192, 48}, {208, 48},
+		{0, 64},  {16, 64},  {32, 64}, {48, 64}, {64, 64}, {80, 64}, {96, 64}, {112, 64}, {128, 64}, {144, 64},
+	}, {
+	       4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,         4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,         4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,
+		   4,      4,         4,        4,        4,        4,        4,        4,         4,         4,
+	}
+};
+
+const is31_led g_is31_indicator_leds[3] = {
```

Comment:
Is there a reason that this is here, and not part of the rgb matrix feature.  Especially as it has "indicator" flags explicitly for this purpose?

Reply:
Updated whole rev_0100.c, have removed none used code.